relayer/middleware/wallet/wallet.middleware.ts

```diff
@@ -106,6 +106,9 @@ export interface WalletOpts {
   privateKeys: Partial<{
     [k in ChainId]: any[];
   }>;
+  tokenAddresses: Partial<{
```

Comment:
this should be optional since it's not always necessary to monitor token balances (some relayers will and some relayers won't)

Reply:
(this will force you to make the `tokenAddresses` paramenter of `startWalletManagement` optional as well

Reply:
you're right, I'll change that

Reply:
done